chat-widget/src/components/webchatcontainerstateful/webchatcontroller/middlewares/renderingmiddlewares/activityMiddleware.tsx

```diff
@@ -79,7 +79,7 @@ export const createActivityMiddleware = (systemMessageStyleProps?: React.CSSProp
         }
 
         if (isTagIncluded(card, Constants.systemMessageTag)) {
-            return handleSystemMessage(next, args, card, systemMessageStyleProps);
+            return handleSystemMessage(next, args, card, renderMarkdown, systemMessageStyleProps);
```

Comment:
as part of the text, we need to corroborate the markdown can be turn it on and off during chat session,

Reply:
Do I need to use renderMarkdownMethod as callback function as written in https://github.com/microsoft/omnichannel-chat-widget/blob/635e2559f838936a3729eaf3718818669fca1d7d/chat-widget/src/components/footerstateful/downloadtranscriptstateful/DownloadTranscriptStateful.tsx#L68

Reply:
as long as when renderMarkdown is passed as dissabled , markdown rendering should not be applied

Reply:
Yes we have passed renderMarkdown as function reference  and when markdown is disabled markdown rendering is not getting applied.